.ado/templates/apple-nuget-publish.yml

```diff
@@ -99,4 +99,3 @@ steps:
       command: push
       packagesToPush: '$(Build.ArtifactStagingDirectory)/Microsoft.FluentUI.ReactNative.*.nupkg'
       publishVstsFeed: Office
-
```

Comment:
Intentional?

Reply:
Prettier (our linter) won't let me add it back. Not intentional but probably harmless